src/NServiceBus.Persistence.AzureStorage/Config/ThrowIfNoCloudTableClientForSubscriptionsProvider.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace NServiceBus.Persistence.AzureStorage
+{
+    using System;
+    using Microsoft.Azure.Cosmos.Table;
+
+    class ThrowIfNoCloudTableClientForSubscriptionsProvider : IProvideCloudTableClientForSubscriptions
+    {
+        public CloudTableClient Client => throw new Exception(
+            $"No CloudTableClient has been configured. Either provide a connection string, use `persistence.UseCloudTableClient(client)` or register an implementation of `{nameof(IProvideCloudTableClient)}` in the container.");
```

Comment:
I wonder if we should change the order in the 'Either provide....' sentence to mention the connection string option the last.